Makefile

```diff
@@ -48,6 +48,9 @@ build: get
 	env GOOS=linux   GOARCH=arm64 go build $(GO_LDFLAGS) -o bin/linux/vcert_arm       ./cmd/vcert
 	env GOOS=linux   GOARCH=amd64 go build $(GO_LDFLAGS) -o bin/linux/vcert           ./cmd/vcert
 	env GOOS=linux   GOARCH=386   go build $(GO_LDFLAGS) -o bin/linux/vcert86         ./cmd/vcert
+	env GOOS=linux   GOARCH=arm GOARM=5 go build $(GO_LDFLAGS) -o bin/linux/vcert_arm32v5       ./cmd/vcert
```

Comment:
Is it possible that you do these changes in a separated PR?

Reply:
This definitely should go in another MR, since is not mentioned in the current description. Adding support to new type of binaries, will increase the MR dependency in another changes, as we need to internal changes in our pipeline along with this changes to be able to actually deploy these new binaries, when another VCert version that contains this changes is released.